chartiq/development/js/standard.js

```diff
@@ -7473,7 +7473,7 @@ CIQ.ChartEngine.prototype.findHighlights = function (isTap, clearOnly) {
 		this.setMeasure();
 	}
 
-	if (highlightedDraggable && !myPanel.noDrag) {
+	if (highlightedDraggable && myPanel && !myPanel.noDrag) {
```

Comment:
you can use optional chaining instead of &&
ref:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

```suggestion
	if (highlightedDraggable && !myPanel?.noDrag) {
```